resources/views/setup/user.blade.php

```diff
@@ -1,15 +1,14 @@
 @extends('layouts/setup')
-{{-- TODO: Translate --}}
-{{-- Page title --}}
+{{ trans('admin/user/table.createuser') }}
 @section('title')
-Create a User ::
+{{ trans('admin/user/general.create_user') }} ::
 @parent
 @stop
 
 {{-- Page content --}}
 @section('content')
 
-<p> This is the account information you'll use to access the site for the first time.  </p>
+<p>{{ trans('admin/user/general.create_user_page_explaination') }}</p>
```

Comment:
This is a small detail, but I think there's a typo in this key - "explaination" vs "explanation"